### PR TITLE
Allow updating multiple plugins at once.

### LINKED ISSUE #10488: several plugin updates available: "update all"-Button
When there are several plugin updates available, it would be handy to not have to do the update procedure step by step for each available update but all in one with a Button/Link "update all"

at the end of the procedure there should be summary shown e.g. "x plugins were updated"

### PR DESCRIPTION
**Pull request by diosmosis**
Simple change, jquery only.

## Screenshots

Update view:

image

After update:

image

Fixes #10488

### CODE REVIEW

**Comment by sgiehl** [comment]
@diosmosis  If someone has a premium plugin installed but the license is missing or expired it now shows:
image
Guess in this case the checkboxes for such plugins should be disabled as it would result in an error...

**Comment by diosmosis** [comment]
@sgiehl disabled the checkboxes in that case

File: plugins/CorePluginsAdmin/lang/en.json [hunk 1]
```diff
@@ -83,6 +83,7 @@
         "Version": "Version",
         "ViewAllMarketplacePlugins": "View all Marketplace plugins",
         "WeCouldNotLoadThePluginAsItHasMissingDependencies": "The plugin %1$s could not be loaded as it has missing dependencies: %2$s",
-        "Websites": "Websites"
+        "Websites": "Websites",
+        "UpdateSelected": "Updated Selected"
```
Comment (sgiehl) [inline plugins/CorePluginsAdmin/lang/en.json]: guess that should be `Update selected`

**Review by sgiehl — CHANGES_REQUESTED** [review]

**Comment by diosmosis** [comment]
Fixed the typo and merged w/ 3.x-dev

File: plugins/CorePluginsAdmin/lang/en.json [hunk 2]
```diff
@@ -83,6 +83,7 @@
         "Version": "Version",
         "ViewAllMarketplacePlugins": "View all Marketplace plugins",
         "WeCouldNotLoadThePluginAsItHasMissingDependencies": "The plugin %1$s could not be loaded as it has missing dependencies: %2$s",
-        "Websites": "Websites"
+        "Websites": "Websites",
+        "UpdateSelected": "Updateg Selected"
```
Comment (sgiehl) [inline plugins/CorePluginsAdmin/lang/en.json]: Now it's another typo 🙈

**Review by sgiehl — CHANGES_REQUESTED** [review]

File: plugins/CorePluginsAdmin/lang/en.json (hunk 2 above)
Comment (diosmosis) [reply plugins/CorePluginsAdmin/lang/en.json]: argh, i think that was part of the git command to commit :)

**Comment by sgiehl** [comment]
@diosmosis  once the typo is fixed, it's good to merge

**Pull request merged**